### ISSUE TITLE
gr-nrsc5 fails on gr3.8 with...

### ISSUE THREAD

**Issue by BM45** [argilo/gr-nrsc5#11]
gr::buffer::allocate_buffer: warning: tried to allocate
   4 items of size 146176. Due to alignment requirements
   16 were allocated.  If this isn't OK, consider padding
   your structure to a power-of-two bytes.
   On this platform, our allocation granularity is 4096 bytes.
Press Enter to quit:

**Comment by argilo** [comment]
@BM45 That's not a failure, it's just a warning. But i'll leave this issue open for now, since I should probably consider whether it's worth adding some padding to silence the warning.

**Comment by BM45** [comment]
You're right, its just a warning! Is the tx flowgraph tested successfully on real(!) hd-radio hardware ? 
Have some issues on a HDR-1 from Sangean. Analog fm signal in multiplex is okay, it can be demodulate successfully. Also the two digital sidebands are shown on the spectrum analyzer, but there is no detection/decoding on real hd-radio device. Arrrr....

**Comment by argilo** [comment]
Yes, I've tested it (in GNU Radio 3.8) with a couple real HD Radio receivers. If the sample flow graphs I provided still aren't working for you, please provide whatever details you can to help me reproduce the problem. Thanks!

**Comment by RobertSwirsky** [comment]
FYI: I was able to receive "maint3-8" release just fine on a Sangaen HDF-16 receiver
